modules/alternatives-to-storing-admin-secrets-in-kube-system.adoc

```diff
@@ -1,14 +1,44 @@
 // Module included in the following assemblies:
```

Comment:
These changes have been reviewed in the other PR for 4.7 CCO work, but I copied them over since that one isn't merged yet.